curriculum/challenges/english/03-front-end-development-libraries/react/create-a-component-with-composition.md

```diff
@@ -28,7 +28,7 @@ When React encounters a custom HTML tag that references another component (a com
 
 In the code editor, there is a simple functional component called `ChildComponent` and a class component called `ParentComponent`. Compose the two together by rendering the `ChildComponent` within the `ParentComponent`. Make sure to close the `ChildComponent` tag with a forward slash.
 
-**Note:** `ChildComponent` is defined with an ES6 arrow function because this is a very common practice when using React. However, know that this is just a function. If you aren't familiar with the arrow function syntax, please refer to the JavaScript section.
+**Note:** `ChildComponent` is defined with an ES6 arrow function because this is a very common practice when using React. However, know that this is just a function. If you aren't familiar with the arrow function syntax, please refer to the [Use Arrow Functions to Write Concise Anonymous Functions](https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/es6/use-arrow-functions-to-write-concise-anonymous-functions) module.
```

Comment:
```suggestion
**Note:** `ChildComponent` is defined with an ES6 arrow function because this is a very common practice when using React.
```

I would prefer to slim this down. I don't think this large explanation or link is necessary. It's expected that campers know the material leading up to whatever challenge they are on - so they should know the arrow function syntax at this point. They haven't seen it with react though, so I'm fine leaving a small note.